lib/control.rb

```diff
@@ -0,0 +1,112 @@
+class Control
+  require_relative './plateu'
+  require_relative './rover'
+  require_relative './houston_command'
+  attr_accessor :x, :y, :plateu, :rover, :commands, :command_array
+
+  def command_rover(string)
```

Comment:
To clarify this whole class, one approach would be to focus on the main engine and let things fall out of that.  eg clarify what the main process is for handling commands.  eg this would probably involve steps like reading in the commands, iterating through the rovers running the commands for each one, printing the state and so on.  This will probably make it easier to pull responsibilities out of this class.